README.md

```diff
@@ -147,9 +147,10 @@ Summary
 
 ## Installation
 
+
 - From source:
   - install a recent Rust toolchain, libx11 and libxrandr (depending on your
-    distribution, you may need to install development headers separately)
+    distribution, you may need to install development headers separately). On Ubuntu you will need to have               `libxrandr-dev` installed. To install it with apt run `apt install libxrandr-dev`
```

Comment:
Actually this is completely redundant (see my previous comment). I really don't want to clutter the readme with distro-specific information, only links to actual packages. Please ask the Ubuntu folks (or better yet, Debian) to package shotgun instead.

Reply:
Once the package exists, I'll gladly accept a PR documenting that.